### DISCUSSION TITLE
Need help with IDE setup

### DISCUSSION THREAD

**Discussion by Oleasdev** [freeplane/freeplane#1820] [Help]
I wanted to setup android studio for the development.
That should be a problem, right?

Now I want to add a run/debug configuration, but I don't know what the main class is:
![Screenshot 2024-05-05 153629](https://github.com/freeplane/freeplane/assets/113558290/579c5913-ac2b-4048-b5ce-72bde1a58adc)

I saw no other topic on this, so i opend a new discussion.
Let's hope that's correct 🙏

**Comment by macmarrum** [comment]
You can have a look at
- https://github.com/freeplane/freeplane/discussions/1321
